crawler/src/main/resources/bin/RetrieveAVRISNGData.py

```diff
@@ -0,0 +1,84 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ftplib import FTP
+import os, sys, os.path
+
+def handleDownload(block):
+    file.write(block)
+    print (".")
+
+
+ddir='/Users/Alex/Documents' # set to local path where you want files downloaded to
```

Comment:
You cannot download the data to this directory... it does not exist on my machine (or anyone else's machine for that matter) other than your own.

Reply:
Actually, I would just make this directory ```/usr/local/coal-sds-deploy/data/staging```